test/ExecutionInfoMapperTests.cs

```diff
@@ -4,6 +4,7 @@
 using Gauge.Dotnet.Wrappers;
 using Gauge.CSharp.Lib;
 using System.Linq;
+using System.Collections.Generic;
```

Comment:
```suggestion
```

Reply:
'using System.Collections.Generic;' is needed